lte/gateway/fabfile.py

```diff
@@ -794,13 +793,6 @@ def _build_magma():
         run('make')
 
 
-def _run_sudo_python_unit_tests():
-    """ Run the magma unit tests """
-    with cd(AGW_ROOT):
-        # Run all unit tests that are not run as pre-commit checks in CI
-        run('make test_sudo_python')
```

Comment:
Is this make target still used anywhere now? If not could we get rid of it?

Reply:
I think until we decide to deprecate make completely, it should still be possible to run these tests manually.

Reply:
I created a ticket for after the switchover https://github.com/magma/magma/issues/14265